config/webpack/webpack.common.js

```diff
@@ -185,7 +186,11 @@ const webpackConfig = ({envFile = '.env', platform = 'web'}) => ({
             },
             {
                 test: /\.css$/i,
-                use: ['style-loader', 'css-loader'],
+                use: ['style-loader', 'file-loader', 'css-loader'],
```

Comment:
NABish but are you sure that this change is needed? I don't see why it would be, and `file-loader` should be prettymuch deprecated nowadays (we probably should've nixed it back when we switched to webpack 5)

Reply:
I can remove it and see if it still works 👍

Reply:
thx